apps/app/src/components/plans/PlanUpgradeModal.tsx

```
@@ -68,6 +71,55 @@ export const PlanUpgradeModal: React.FC<PlanUpgradeModalProps> = ({
+    return (
+      <div className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50 p-4">
+        <div className="bg-white dark:bg-gray-900 rounded-lg max-w-md w-full">
+          <div className="flex items-center justify-between p-6 border-b">
+            <div className="flex items-center gap-3">
+              <Key className="w-5 h-5 text-purple-600" />
+              <h2 className="text-xl font-bold">License Required</h2>
+            </div>
+            <button
+              onClick={onClose}
+              className="text-gray-400 hover:text-gray-600"
+            >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)